### ISSUE TITLE
feat(patch): Shared patch to trust user certificates

### ISSUE THREAD

**Issue by bennett-sh** [revanced/revanced-patches#1813]
### Application

All apps

### Issue

It's interesting to inspect HTTPS traffic but most apps only trust system and not user certificates making it much harder.

### Patch

1. Does the manifest's application tag have `android:networkSecurityConfig="@xml/SOMETHING"`?
|> If not, create `xml/network_security_config.xml` and add it to the manifest.
```xml
<?xml version="1.0" encoding="utf-8"?>
<network-security-config>
  <base-config>
      <trust-anchors>
          <certificates src="system" />
          <certificates src="user" overridePins="true" />
      </trust-anchors>
  </base-config>
</network-security-config>
```
|> If yes, add the ```<certificates src="user" overridePins="true" />```-element

### Motivation

My motivation is that it improves the work of many people by speeding up app patching and making HTTPS interception without root easier.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by proxykingdev** [comment]
simply adding this XML will not solve your problem.

**Comment by Axelen123** [comment]
[apk-mitm](https://github.com/shroudedcode/apk-mitm) already exists

**Comment by bennett-sh** [comment]
yes but it makes everything more complicated since I'll have to go through a secondary step on my pc (or termux; unsure if it works) everytime I want to update an app
